src/test/java/seedu/address/testutil/TypicalPersons.java

```diff
@@ -56,10 +62,16 @@ public class TypicalPersons {
     // Manually added - Person's details found in {@code CommandTestUtil}
     public static final Person AMY = new PersonBuilder().withName(VALID_NAME_AMY).withPhone(VALID_PHONE_AMY)
             .withEmail(VALID_EMAIL_AMY).withAddress(VALID_ADDRESS_AMY).withBirthday(VALID_BIRTHDAY_AMY)
-            .withTags(VALID_TAG_FRIEND).build();
+            .withMatriculationYear(VALID_MATRICULATIONYEAR_AMY).withTags(VALID_TAG_FRIEND).build();
```

Comment:
Perhaps you could update your variable names to comply with the coding standard?

Reply:
Thanks for the suggestion! I have updated the code accordingly with the suggested changes.